operations_test.go

```diff
@@ -636,3 +644,78 @@ func TestKillProcessErrors(t *testing.T) {
 		})
 	}
 }
+
+
+func TestSwitchControlProcessor(t *testing.T) {
+	initOperationFakes(t)
+	var got string
+	fakeGNOI.SCPer = func(ctx context.Context, in *spb.SwitchControlProcessorRequest, 
+													opts ...grpc.CallOption) (*spb.SwitchControlProcessorResponse, error) {
+		got = getProcessorName(in.ControlProcessor)
+		resp := spb.SwitchControlProcessorResponse{
+			ControlProcessor : in.ControlProcessor,
+		}
+
+		return &resp, nil
+	}
+
+	cisco_dut := DUT(t,"dut_cisco")
```

Comment:
Go style is camelCase for variable names. Fine for the DUT to be named "dut_cisco", but the variable would have to be ciscoDUT.